client/src/components/dashboard/EditProfileForm.js

```diff
@@ -38,7 +39,13 @@ function EditProfileForm() {
 
   // Get dispatch method and state from auth context
   const dispatch = useContext(AuthDispatchContext);
-  const { user, profile } = useContext(AuthStateContext);
```

Comment:
in my merge I removed user and replaced it with profile. So instead of user._id it's profile._id that is being used